webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'patternfly-react';
+import classNames from 'classnames';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './StoppedTasksCard.scss';
+
+class StoppedTasksCard extends React.Component {
+  getTableItemClassName = (item, type) => {
+    const { focusedOn } = this.props;
+    return focusedOn.normal ||
+      focusedOn.total ||
+      (focusedOn[item] && focusedOn[item][type])
+      ? `${type}-data`
+      : 'not-focused';
+  };
+  shouleBeSelected = focusedOn =>
+    focusedOn &&
+    (focusedOn.total ||
+      ((focusedOn.warning &&
+        (focusedOn.warning.total || focusedOn.warning.last)) ||
+        (focusedOn.success &&
+          (focusedOn.success.total || focusedOn.success.last))) ||
+      (focusedOn.error && (focusedOn.error.total || focusedOn.error.last)));
+  render() {
+    const {
+      className,
+      error,
+      warning,
+      success,
+      timePeriod,
+      focusedOn,
+      onTitleClick,
+    } = this.props;
+    const data = { error, warning, success };
+
+    return (
+      <Card
+        className={classNames(
+          'tasks-donut-card',
+          'stopped-tasks-card',
+          className,
+          {
+            'selected-tasks-card': this.shouleBeSelected(focusedOn),
+          }
+        )}
+      >
+        <Card.Title onClick={onTitleClick}>{__('Stopped')}</Card.Title>
+        <Card.Body>
+          <table className="table table-titles">
+            <tbody>
+              <tr>
+                <td />
+                <td>Total</td>
+                <td>{timePeriod}</td>
+              </tr>
+            </tbody>
+          </table>
+          <table className="table table-bordered table-striped">
+            <tbody>
+              {Object.keys(data).map((item, index) => (
+                <tr className={`${item}-row`} key={index}>
+                  <td>{item[0].toUpperCase() + item.slice(1)}</td>
+                  <td
+                    className={this.getTableItemClassName(item, 'total')}
+                    onClick={data[item].total.onClick}
+                  >
+                    {data[item].total.value}
+                  </td>
+                  <td
+                    className={this.getTableItemClassName(item, 'last')}
+                    onClick={data[item].last.onClick}
+                  >
+                    {data[item].last.value}
+                  </td>
+                </tr>
+              ))}
+            </tbody>
+          </table>
+        </Card.Body>
+      </Card>
+    );
+  }
+}
+
+StoppedTasksCard.propTypes = {
+  className: PropTypes.string,
+  focusedOn: PropTypes.shape({
+    normal: PropTypes.bool,
```

Comment:
Are you missing the `total` prop?

imo you should have `focusedOn.mode: PropTypes.oneOf(['TOTAL', 'NORMAL', 'NONE'])`, can be reused from the `schedule` card

Reply:
It was missing, thanks.
Cant use only these 3 states because there are 6 additional states to focusedOn